package.json

```diff
@@ -35,26 +35,29 @@
     ]
   },
   "scripts": {
-    "lint": "eslint *.js settings",
+    "start": "stripes serve",
+    "lint": "eslint .",
     "test": "echo 'placeholder. no tests implemented'"
   },
   "devDependencies": {
-    "babel-core": "^6.17.0",
-    "babel-eslint": "^8.0.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.16.0",
-    "babel-preset-stage-0": "^6.16.0",
-    "babel-register": "^6.18.0",
-    "eslint": "^4.8.0",
     "@folio/eslint-config-stripes": "^3.0.0",
-    "webpack": "1.11.0"
+    "@folio/stripes-cli": "^1.4.0",
+    "@folio/stripes-connect": "^3.1.0",
+    "@folio/stripes-core": "^2.7.0",
+    "@folio/stripes-logger": "^0.0.2",
+    "babel-eslint": "^9.0.0",
+    "eslint": "^5.5.0",
+    "react": "^16.5.0",
+    "react-dom": "^16.5.0",
+    "react-redux": "^5.0.7",
+    "redux": "^4.0.0",
+    "webpack": "^4.10.2"
```

Comment:
Thanks for the clean-up!  We can remove Webpack as well.

Reply:
`eslint-import-resolver-webpack` wants a `webpack`:
`warning "@folio/eslint-config-stripes > eslint-import-resolver-webpack@0.10.1" has unmet peer dependency "webpack@>=1.11.0".`

Reply:
Odd. If true, then every app, by use of eslint-config-stripes, needs a webpack dependency.  It should be getting webpack from stripes-cli/stripes-core, no?

Reply:
Technically, yes, `eslint-import-resolver-webpack` would still work without that `"webpack": "^4.10.2"` `devDependencies` line (because `stripes-core` needed a `webpack`), but NPM sees that as anti-pattern and will warn.